runtime-watcher/tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,233 @@
+package e2e_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/runtime-watcher/skr/tests/e2e/utils"
+	apicorev1 "k8s.io/api/core/v1"
+
+	apiappsv1 "k8s.io/api/apps/v1"
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	controlPlaneNamespace = "kcp-system"
+	watcherCrName         = "klm-kyma-watcher"
+	remoteNamespace       = "kyma-system"
+	localHostname         = "0.0.0.0"
+	k3dHostname           = "host.k3d.internal"
+)
+
+type ResourceName = types.NamespacedName
+
+var errWatcherDeploymentNotReady = errors.New("watcher Deployment is not ready")
+
+var _ = Describe("Enqueue Event from Watcher", Ordered, func() {
+	kyma := utils.NewKymaWithSyncLabel("kyma-sample", "kcp-system", "regular",
```

Comment:
At least one of these magic strings can be replaced with a constant declared above. I suggest making them all constants.